scripts/lib.sh

```diff
@@ -109,6 +109,8 @@ install-py-dev() {
         pip install https://storage.googleapis.com/python-arm64-wheels/psycopg2_binary-2.8.6-cp38-cp38-macosx_11_0_arm64.whl
         # The CPATH is needed for confluent-kakfa --> https://github.com/confluentinc/confluent-kafka-python/issues/1190
         export CPATH="$(brew --prefix librdkafka)/include"
+        export C_INCLUDE_PATH="$(brew info librdkafka | sed -n 4p | cut -d' ' -f1)/include/"
+        export LIBRARY_PATH="$(brew info librdkafka | sed -n 4p | cut -d' ' -f1)/lib"
```

Comment:
this shouldn't be needed -- my guess is `make bootstrap` skips this part entirely (the `CPATH` already adds the include directory) -- also `brew info | sed` is very fragile -- and the entire reason `brew --prefix` exists!

Reply:
`brew --prefix` gave me a different result. I wasn't too sure which one I actually needed. I wasn't able to get it to work with `brew --prefix` on my system, but I'll retry it.

```
❯ brew --prefix librdkafka
/opt/homebrew/opt/librdkafka
```

```
❯ echo $C_INCLUDE_PATH
/opt/homebrew/Cellar/librdkafka/1.9.2/include/
```

Reply:
yeah and `readlink -f /opt/homebrew/opt/librdkafka` will tell you the rest of the story

Reply:
well, readlink doesn't work on macos -- but you get the idea -- brew uses symlinks into `Cellar`